packages/components/input/src/Input.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import { FormField } from '@spark-ui/form-field'
+import { Icon } from '@spark-ui/icon'
+import { Check } from '@spark-ui/icons/dist/icons/Check'
+import { PenOutline } from '@spark-ui/icons/dist/icons/PenOutline'
+import { VisuallyHidden } from '@spark-ui/visually-hidden'
+import { Meta, StoryFn } from '@storybook/react'
+import { ChangeEvent, useState } from 'react'
+
+import { Input, InputGroup, InputProps } from '.'
+
+const meta: Meta<typeof Input> = {
+  title: 'Experimental/Input',
+  component: Input,
+}
+
+export default meta
+
+export const Default: StoryFn = _args => <Input />
+
+export const Uncontrolled: StoryFn = _args => <Input defaultValue="IPhone 12" />
+
+export const Controlled: StoryFn = () => {
+  const [value, setValue] = useState('IPhone 13')
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value)
+  }
+
+  return <Input value={value} onChange={handleChange} />
+}
+
+export const Disabled: StoryFn = _args => <Input defaultValue="IPhone" disabled />
+
+const intents: InputProps['intent'][] = ['neutral', 'success', 'alert', 'error']
+
+export const Intent: StoryFn = _args => {
+  return (
+    <div className="gap-md flex flex-col">
+      {intents.map(intent => (
+        <Input key={intent} intent={intent} />
+      ))}
+    </div>
+  )
+}
+
+export const GroupAddons: StoryFn = _args => {
+  return (
+    <InputGroup>
+      <InputGroup.LeftAddon>https://</InputGroup.LeftAddon>
+      <Input />
+      <InputGroup.RightAddon>.com</InputGroup.RightAddon>
+    </InputGroup>
+  )
+}
+
+export const GroupElements: StoryFn = _args => {
+  return (
+    <InputGroup>
+      <InputGroup.LeftElement>
+        <Icon>
+          <PenOutline />
+        </Icon>
+      </InputGroup.LeftElement>
+
+      <Input />
+
+      <InputGroup.RightElement>
+        <Icon>
+          <Check />
+        </Icon>
+      </InputGroup.RightElement>
+    </InputGroup>
+  )
+}
+
+export const GroupDisabled: StoryFn = _args => (
+  <InputGroup isDisabled>
+    <InputGroup.LeftAddon>https://</InputGroup.LeftAddon>
+    <Input defaultValue="adevinta.com" />
+    <InputGroup.RightElement>
+      <Icon>
+        <Check />
+      </Icon>
+    </InputGroup.RightElement>
+  </InputGroup>
+)
+
+export const FieldLabel: StoryFn = _args => {
+  return (
+    <FormField className="!gap-sm" name="title">
```

Comment:
Why important? `!gap-sm`

Reply:
Because I overwriting the gap and since we are not using tailwind merge this is the recommended way for now